cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4B" NARROW []
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity third quarter
+def: "Fraction of identified MS2 in the third quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4C" NARROW []
+
+[Term]
+id: QC:40000
+name: Explained precursor intensity fourth quarter
+def: "Fraction of identified MS2 in the fourth quarter of precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4D" NARROW []
+
+[Term]
+id: QC:40000
+name: Extent of identified precursor intensity
+def: "Ratio of 95th over 5th percentile of precursor intensity for identified peptides" [PSI:QC]
+comment: Can be used to approximate the dynamic range of signal
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+has_units: STATO:0000300 ! dimensionless ratio
+synonym: "MS1-3A" NARROW []
+
+[Term]
+id: QC:40000
+name: Precursor intensity range of identified MS2
+def: "Minimum and maximum precursor intensity recorded and identified." [PSI:QC]
+comment: The intensity range of the identified precursors informs about the dynamic range of the acquisition. 
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: MS:1003085 ! previous MSn-1 scan precursor intensity
+
+[Term]
+id: QC:40000
+name: Precursor intensity of identified MS2 Q1, Q2, Q3
+def: "From the distribution of precursor intensity of identified MS2, the quartiles Q1, Q2, Q3" [PSI:QC] 
+comment: The (un)identified precursor intensity distribution can aid the interpretation of overall identification success.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: STATO:0000167 ! first quartile, STATO:0000574 ! center value, STATO:0000170 ! third quartile
+
+[Term]
+id: QC:40000
+name: Precursor intensity of unidentified MS2 Q1, Q2, Q3
+def: "From the distribution of precursor intensity of unidentified MS2, the quartiles Q1, Q2, Q3" [PSI:QC] 
+comment: The (un)identified precursor intensity distribution can aid the interpretation of overall identification success.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: STATO:0000167 ! first quartile, STATO:0000574 ! center value, STATO:0000170 ! third quartile
+
+[Term]
+id: QC:40000
+name: Median S/N for MS1 spectra in RT range in which half of the peptides are identified
```

Comment:
how is this 'half' computed? There are a lot of RT ranges which fulfill this property....

Reply:
True, there is a metric later that uses the shortest RT period with 50%. This here is Q1 to Q3, I'll add more descr.